File: Polypaint/PolyPaint/PolyPaint/Modeles/Stats.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace PolyPaint.Modeles
{
    class Stats
    {
        public int nbGames { get; set; }
        public int nbVictories { get; set; }
        public double victoryPercentage { get; set; }
        public double averageMatchesTime { get; set; }
        public double totalMatchesTime { get; set; }

        public Stats(int nbGames, int nbVictories, double victoryPercentage, double averageMatchesTime, double totalMatchesTime)
        {
            this.nbGames = nbGames;
            this.nbVictories = nbVictories;
            this.victoryPercentage = victoryPercentage;
            this.averageMatchesTime = averageMatchesTime;
            this.totalMatchesTime = totalMatchesTime;
        }
    }
}

File: Polypaint/PolypaintApp/app/src/main/java/com/example/polypaintapp/Stats.kt
package com.example.polypaintapp

class Stats {

    constructor(nbGames: Number, nbGamesCoop: Number, nbGamesSolo: Number, nbVictories: Number, victoryPercentage: Number, averageMatchesTime: Number,
                totalMatchesTime: Number) {
        this.nbGames = nbGames
        this.nbGamesCoop = nbGamesCoop
        this.nbGamesSolo = nbGamesSolo
        this.nbVictories = nbVictories
        this.victoryPercentage = victoryPercentage
        this.averageMatchesTime = averageMatchesTime
        this.totalMatchesTime = totalMatchesTime
    }

    var nbGames: Number
    var nbGamesCoop: Number
    var nbGamesSolo: Number
    var nbVictories: Number
    var victoryPercentage: Number
    var averageMatchesTime: Number
    var totalMatchesTime: Number
}